.github/workflows/pull_request_check.yml

```diff
@@ -0,0 +1,71 @@
+name: Perform unit test on the new pull requests.
+
+on:
```

Comment:
Let's also include `workflow_dispatch` trigger so we can run each action manually too.

Reply:
Okay, I will add it.

Reply:
I have added the `workflow_dispatch` trigger too. However, in the Actions tab, as indicated in their documentation, I can't seem to find the `run Workflow` option even after adding the trigger.

Reply:
I figured the issue with the `workflow_dispatch` event not showing the ability to run from the `Actions` tab. Apparently the workflow should be on the `default` branch for it to give us the ability to run from the `Actions` tab.